en/installconfig-upgrade-postgres.md

```diff
@@ -1,109 +1,115 @@
-Title: Upgrade from <=2.3 (Xenial) to 2.4 (Bionic)
+Title: Upgrade 2.3 to 2.4 from Ubuntu 16.04
 
-# Upgrade from <= (Xenial) to 2.4 (Bionic)
+# Upgrade 2.3 to 2.4 from Ubuntu 16.04
 
-MAAS 2.3 is the last releases support on Ubuntu 16.04 LTS (Xenial Xerus). This is
-because 18.04 changes the base dependencies, which require MAAS to make changes
-that are not backportable to Ubuntu 16.04 LTS (Xenial Xerus).
+MAAS 2.3 is the last supported version of MAAS for Ubuntu 16.04 LTS (Xenial
+Xerus). This is because changes in the base dependencies of Ubuntu 18.04 LTS
+(Bionic Beaver) require MAAS updates that are not back-portable to
+older versions of MAAS. Consequently, to upgrade from MAAS 2.3 to MAAS 2.4 on Ubuntu
```

Comment:
This whole paragraph seems rather overcomplex to me. Suggest slight rewording:

MAAS 2.3 is the last supported version for Ubuntu 16.04 LTS. Changes in the base dependencies of Ubuntu 18.04 LTS mean that newer versions of MAAS will not be back-portable, and consequently, to upgrade to MAAS 2.4 and all future versions, you will also need to upgrade the base operating system.

Reply:
:+1: thank you!